application/controllers/admin/surveypermission.php

```diff
@@ -541,6 +541,9 @@ function set($surveyid)
      */
     function delete($surveyid)
     {
+        if(!Yii::app()->getRequest()->isPostRequest) {
```

Comment:
Maybe we can place this in a some common library or base controller method?
something like $this->requirePost()

Reply:
yep